client/my-sites/hosting/test/hosting-activate-status.jsx

```diff
@@ -62,20 +62,20 @@ describe( 'index', () => {
 
 		render(
 			<Provider store={ store }>
-				<HostingActivateStatus context="plugins" />
+				<HostingActivateStatus context="plugin" />
```

Comment:
Any reason to change it from plural to singular?

Reply:
the `HostingActivateStatus` component `switch` uses singular, it was causing the test to fail as it always goes to the default case.

Reply:
> After starting the trial I did not see the plugin upload drop area:

This had always worked for me and show the dropzone, I'll figure why it didn't work